FORMAT_JSON.md

```diff
@@ -0,0 +1,128 @@
+# Implementation Details: JSON
+
+## General Notes
+
+JSON is a format build on attribute-value pairs (dictionaries) and arrays.
+It is most commonly represented as text.
+
+Thus, in order to differentiate between openPMD **attributes** and **data sets**, slight naming conventions need to be introduced.
+An example implementation is provided in [openPMD-api](https://github.com/openPMD/openPMD-api) and in doubt, its conventions shall apply.
+
+The following JSON keys (names) are reserved for openPMD usage with JSON:
+- `attributes`
+- `data`
+- `datatype`
+
+## Datatype Conventions
+
+An openPMD **data set** is represented by a JSON object with keys `attributes`, `data` and `datatype`:
+```json
```

Comment:
Maybe eliminate the `json` tag here? `{"attributes": <attributes>, "data": <data>, "datatype": <datatype>}` is no valid JSON and some formatters may not like it.

Comparison with Github's formatter:
```
{"attributes": <attributes>, "data": <data>, "datatype": <datatype>}
```

```json
{"attributes": <attributes>, "data": <data>, "datatype": <datatype>}
```

Reply:
I didn't find this concerning, yet sounds reasonable to avoid future breakage.

For JSON (and XML) we could even create [schemas](http://json-schema.org/understanding-json-schema/index.html) :)